docs/APP-CONNECTIVITY.md

```diff
@@ -0,0 +1,299 @@
+# Application Connectivity
+
+Edge applications often require network connectivity to access other applications or some
+external network endpoints. EVE provides different means of connectivity, varying in properties,
+feature-sets and performance considerations.
+
+## Application network interfaces
+
+Network connectivity is enabled for an application by configuring one or more network adapters.
+These consists of [virtual network interfaces](#virtual-network-interfaces), connecting
+application with [network instances](#network-instances), and directly assigned
+[physical NIC](#physical-network-ports) or [SR-IOV virtual functions](#sr-iov-vfs).
+The order of network adapter configurations inside `AppInstanceConfig` matters, because this
+is the same order at which these network adapters will be presented to the application by VMM.
```

Comment:
Probably should expand the 'VMM', in EVE doc, it can also be 'VM Monitor'.

Reply:
Done